src/test/java/uk/gov/pay/api/it/GetDirectDebitEventsTest.java

```diff
@@ -65,13 +61,13 @@ public void getDirectDebitEvents() {
                 .body("results[0].event_date", is("2018-03-13T10:00:04.666Z"))
                 .body("results[0].event", is("PAYMENT_ACKNOWLEDGED_BY_PROVIDER"))
                 .body("results[0].event_type", is("CHARGE"))
-                .body("results[0]._links.agreement", is("/v1/agreements/1"))
-                .body("results[0]._links.payment", is("/v1/payments/4"))
+                .body("results[0]._links.agreement", is(app.getConfiguration().getBaseUrl() + "v1/agreements/1"))
```

Comment:
Probably is simpler to see what value we test by extracting `app.getConfiguration.getBaseUrl()` into a local variable and reuse it

Reply:
Agreed, I've made this change.